EPlast/EPlast/Views/Shared/_Layout.cshtml

```diff
@@ -30,7 +30,7 @@
 
 </head>
 <body>
-    @if (SignInManager.IsSignedIn(User))
+    @if (!SignInManager.IsSignedIn(User))
```

Comment:
Why you do this?

Reply:
Because we don`t have any page for don`t signed in user, and i needed to see if my code works, i will fix it right now!